dune

```diff
@@ -0,0 +1 @@
+(data_only_dirs monorepo-bench)
```

Comment:
I don't think this works since we have a top level dune-file instead. In any case, I'd like to see this under bench/ if possible.

Reply:
You could exclude it with `(dirs ..)`.

Reply:
I ended up moving the monorepo-bench dir to bench/monorepo and adding `(dirs :standard \ monorepo)` to bench/dune. I don't understand why `(dirs ..)` works. Did you mean that putting `(dirs ..)` in a dune file would exclude that file's directory from the build? Why is it necessary to refer to the parent directory?